invesalius/gui/widgets/canvas_renderer.py

```diff
@@ -621,7 +727,7 @@ def draw_rectangle(self, pos, width, height, line_colour=(255, 0, 0, 128),
         gc.DrawRectangle(px, py, width, -height)
         self._drawn = True
 
-    def draw_text(self, text, pos, font=None, txt_colour=(255, 255, 255)):
+    def draw_text(self, text, pos, font=None, txt_colour=(255, 255, 255)) -> None:
```

Comment:
text is str
pos is tuple of two integers
font is wx.Font
line_colour is tuple of 4 integers

Reply:
I think that pos is a `Tuple[float, float]` not a `Tuple[int, int]`

Reply:
Like I said in previous comment, every position (`pos`, `pos0`, `pos1`, etc) you can use `float`